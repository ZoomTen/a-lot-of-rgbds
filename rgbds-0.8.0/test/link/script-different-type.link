sram 1
	"ROM1"
