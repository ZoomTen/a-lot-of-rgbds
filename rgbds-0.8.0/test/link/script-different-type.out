error: script-different-type.link(2): "ROM1" is specified to be a SRAM section, but it is already a ROMX section
Linking failed with 1 error
